ui/scss/component/section.scss

```diff
@@ -125,6 +125,21 @@
   font-size: var(--font-body);
 }
 
+.section__subtitle--small {
+  @extend .card__subtitle;
+  margin: 0px;
+  font-size: var(--font-xsmall);
+}
```

Comment:
From BEM perspective, a modifier must always modify something, not used standalone.  When the rule is followed, there is no need to `@extend`. There are a few incorrect patterns like this, but let's clean up moving forward.

- Remove `@extend .card__subtitle;`
- At the client side, use `section__subtitle section__subtitle--small`